app/common/lib/suggestion.js

```diff
@@ -540,12 +540,15 @@ const getSearchSuggestions = (state, tabId, urlLocationLower) => {
 
 const getAlexaSuggestions = (state, urlLocationLower) => {
   return new Promise((resolve, reject) => {
+    const options = {
```

Comment:
Please follow the same pattern as here:
https://github.com/luixxiul/browser-laptop/blob/dd9ad5fe6077165d5d15df2428c5770a235f04d1/app/common/lib/suggestion.js#L504

Basically you resolve with an empty immutable list if the setting is not on.